modules/overviews/app/components/overviews/project_custom_fields/item_component.rb

```diff
@@ -42,10 +42,67 @@ def initialize(project_custom_field:, project_custom_field_values:, project:)
         @project_custom_field = project_custom_field
         @project_custom_field_values = project_custom_field_values
         @project = project
+
+        # Set up tooltip for calculated fields
+        if @project_custom_field.calculated_value?
+          @tooltip_id = "calculated-field-tooltip-#{@project_custom_field.id}"
+          @tooltip = Primer::Alpha::Tooltip.new(
+            for_id: @tooltip_id,
+            type: :description,
+            text: I18n.t("custom_fields.calculated_field_not_editable"),
+            direction: :s
+          )
+        end
       end
 
       private
 
+      def allowed_to_edit?
+        !@project_custom_field.calculated_value? &&
+          User.current.allowed_in_project?(:edit_project_attributes, @project)
+      end
+
+      def authorized_edit_wrapper
+        if allowed_to_edit?
+          Primer::Beta::Text.new(
+            tag: :div,
+            classes: "project-custom-field-clickable",
+            data: {
+              controller: "project-custom-field-edit async-dialog",
+              "project-custom-field-edit-url-value": edit_project_custom_field_path(project_id: @project.id,
+                                                                                    id: @project_custom_field.id),
+              action: "click->project-custom-field-edit#openEditDialog " \
+                      "keydown.enter->project-custom-field-edit#openEditDialog " \
+                      "keydown.space->project-custom-field-edit#openEditDialog " \
+                      "project-custom-field-edit:open-dialog->async-dialog#handleOpenDialog"
+            },
+            aria: {
+              label: [
+                I18n.t(:label_edit_x, x: @project_custom_field.name),
+                I18n.t(:label_value_x, x: accessible_value_text)
+              ].join(", ")
+            },
+            role: "button",
+            tabindex: 0,
+            test_selector: "project-custom-field-edit-button-#{@project_custom_field.id}"
+          )
+        else
+          Primer::Beta::Text.new(
+            tag: :div,
+            id: @tooltip_id,
```

Comment:
This instance variable will only be set if the cf is a calculated value, but the code does not suggest that. Maybe using a method called e.g. `calculated_value_tooltip_id` makes it clearer.